### ISSUE TITLE
Error message when compiler is not present could be improved

### ISSUE THREAD

**Issue by shenki** [linuxppc/issues#82]
```
$ make
make: powerpc64le-linux-gnu-asfasfdagcc: Command not found
./scripts/gcc-version.sh: line 25: powerpc64le-linux-gnu-asfasfdagcc: command not found
./scripts/gcc-version.sh: line 26: powerpc64le-linux-gnu-asfasfdagcc: command not found
./scripts/gcc-version.sh: line 25: powerpc64le-linux-gnu-asfasfdagcc: command not found
./scripts/gcc-version.sh: line 26: powerpc64le-linux-gnu-asfasfdagcc: command not found
./scripts/gcc-version.sh: line 25: powerpc64le-linux-gnu-asfasfdagcc: command not found
./scripts/gcc-version.sh: line 26: powerpc64le-linux-gnu-asfasfdagcc: command not found
./scripts/gcc-version.sh: line 28: powerpc64le-linux-gnu-asfasfdagcc: command not found
Sorry, GCC v4.0 or above is required to build the 64-bit powerpc kernel.
arch/powerpc/Makefile:358: recipe for target 'checkbin' failed
make: *** [checkbin] Error 1
```

**Comment by chleroy** [comment]
This doesn't seem to be a powerpc issue. That same happens with x86:

[root@po16846vm linux-powerpc]# make CROSS_COMPILE=gcc-is-not-there-   ARCH=x86
/bin/sh: gcc-is-not-there-gcc: command not found
make[1]: gcc-is-not-there-gcc: Command not found
/bin/sh: gcc-is-not-there-gcc: command not found
/bin/sh: gcc-is-not-there-gcc: command not found
scripts/kconfig/conf  --syncconfig Kconfig
./scripts/gcc-version.sh: line 17: gcc-is-not-there-gcc: command not found
./scripts/gcc-version.sh: line 18: gcc-is-not-there-gcc: command not found
./scripts/gcc-version.sh: line 19: gcc-is-not-there-gcc: command not found
./scripts/gcc-version.sh: line 17: gcc-is-not-there-gcc: command not found
./scripts/gcc-version.sh: line 18: gcc-is-not-there-gcc: command not found
./scripts/gcc-version.sh: line 19: gcc-is-not-there-gcc: command not found
./scripts/clang-version.sh: line 11: gcc-is-not-there-gcc: command not found
./scripts/gcc-plugin.sh: line 11: gcc-is-not-there-gcc: command not found
init/Kconfig:16:warning: 'GCC_VERSION': number is invalid

**Comment by mpe** [comment]
Yes you're right. @shenki do you want to report this to the kbuild folks?
